README.zh.md

```
@@ -1,371 +1,142 @@
-| `CODEXMATE_FORCE_RESET_EXISTING_CONFIG` | `0` | 设为 `1` 时首次可强制重建托管配置 |
+## 特别鸣谢
 
-## 技术栈
+感谢所有贡献者对 Codex Mate 的辛勤付出 ❤️
 
-- Node.js
-- Vue.js 3（Web UI）
-- 原生 HTTP Server
-- `@iarna/toml`、`json5`
+<a href="https://github.com/SakuraByteCore/codexmate/graphs/contributors">
+  <img src="https://contrib.rocks/image?repo=SakuraByteCore/codexmate" />
```

markdownlint-cli2 (0.22.1)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)